src/components/BlipGroupCard.vue

```diff
@@ -40,17 +40,31 @@
           <img v-else-if="group.status === 'accepted' && group.position === 'right'" class='accepted' :src="checkSentSvg" draggable="false"/>
           <img v-else-if="group.status === 'received' && group.position === 'right'" class='received' :src="doubleCheckReceivedSvg" draggable="false"/>
           <img v-else-if="group.status === 'consumed' && group.position === 'right'" class='consumed' :src="doubleCheckReadSvg" draggable="false"/>
-          <div v-else-if="group.status === 'failed' && group.position === 'right'" class="failure">
-            {{ failedMessageNotification(group.msgs[0].type) }}
-          </div>
+          <span v-else-if="group.status === 'failed' && group.position === 'right'" class='failure'>
+            <img v-if="onFailedClickIcon" :src="alertSvg" draggable="false"/>
+            <span v-else>
+               {{ failedMessageNotification(group.msgs[0].type) }}
+            </span>
+          </span>
           <span>{{ group.date }}</span>
         </div>
         <div class="flex" :class="'group-notification ' + group.position" v-else>
           <img v-if="group.status === 'dispatched' && group.position === 'right'" :src="clockSvg" draggable="false">
         </div>
       </div>
-      <div :class="'blip-card-photo ' + group.position" v-if="group.photo && group.position === 'right'" :style="{ bottom: '10px', right: '0%', width: '25px', height: '25px', 'background-image': 'url(&quot;' + group.photo + '&quot;)' }">
-      </div>
+      <span v-if="onFailedClickIcon && group.status === 'failed'">
+        <bds-icon v-if="group.position === 'right'"
+          name="info" 
+          theme="solid" 
+          aria-label="Active message failed reason" 
```

Comment:
It's great to see you caring about accessibility. I think you could use translations to define the aria label. What do you think?

```suggestion
          :aria-label="translations.failedIconLabel" 
```

Reply:
Done